src/index.tsx

```diff
@@ -72,7 +71,6 @@ class Root extends PureComponent {
         <ConnectedRouter history={history}>
           <Route component={GetLinks} />
           <Route component={Setup} />
-          <Route component={NotFound} />
```

Comment:
we still need to figure out a way to add this in nicely for when i go to `/cool/kids` instead of `/org/:orgID/cool/kids`. or does this still work in that example?

Reply:
this should work for this example, i got it to work for both types of invalid urls we discussed